src/style.css

```diff
@@ -187,9 +187,9 @@ input {
 .search input {
   width: 70%;
   padding: 0.8vh;
-  padding-left: 3vw;
+  padding-left: 3 vw;
```

Comment:
This is little bit funny. If you do not want to use that left padding, why just not remove it?